### PR TITLE
[MOSIP-35490] moved required helm and deploy scripts to modular repo

### PR DESCRIPTION
**Pull request by ckm007**

### CODE REVIEW

**Review by ashok-ksharma — APPROVED** [review]

File: deploy/minio-client-util/install.sh [hunk 1]
```diff
@@ -0,0 +1,83 @@
+#!/bin/bash
+# Installs minio-client-util
+## Usage: ./install.sh [kubeconfig]
+
+if [ $# -ge 1 ] ; then
+  export KUBECONFIG=$1
+fi
+
+NS=minio-client-util
+CHART_VERSION=1.0.0
```
Comment (Prafulrakhade) [inline deploy/minio-client-util/install.sh]: CHART_VERSION=0.0.1-develop

File: helm/minio-client-util/values.yaml [hunk 2]
```diff
@@ -0,0 +1,320 @@
...
+##
+extraDeploy: []
+
+## Number of nodes
+##
+replicaCount: 1
+
+image:
+  registry: docker.io
+  repository: mosipdev/minio-client-util
```
Comment (Prafulrakhade) [inline helm/minio-client-util/values.yaml]: mosipqa/minio-client-util

File: helm/minio-client-util/values.yaml [hunk 3]
```diff
@@ -0,0 +1,320 @@
...
+  tag: latest
```
Comment (Prafulrakhade) [inline helm/minio-client-util/values.yaml:39]: tag: develop

File: helm/softhsm-backup/values.yaml [hunk 4]
```diff
@@ -0,0 +1,322 @@
...
+##
+extraDeploy: []
+
+## Number of nodes
+##
+replicaCount: 1
+
+image:
+  registry: docker.io
+  repository: mosipdev/softhsm-backup
```
Comment (Prafulrakhade) [inline helm/softhsm-backup/values.yaml]: repository: mosipqa/softhsm-backup

File: helm/softhsm-backup/values.yaml [hunk 5]
```diff
@@ -0,0 +1,322 @@
...
+  tag: latest
```
Comment (Prafulrakhade) [inline helm/softhsm-backup/values.yaml]: tag: develop

**Pull request merged**